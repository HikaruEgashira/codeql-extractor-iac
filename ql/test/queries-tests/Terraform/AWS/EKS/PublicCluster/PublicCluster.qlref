security/Terraform/AWS/EKS/PublicCluster/PublicCluster.ql