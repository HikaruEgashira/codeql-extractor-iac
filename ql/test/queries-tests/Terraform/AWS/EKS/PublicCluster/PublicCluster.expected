| eks.tf:4:3:6:3 | vpc_config | EKS Cluster Publically Accessible |
